engine/src/main/java/org/terasology/persistence/typeHandling/TypeHandlerLibrary.java

```diff
@@ -198,17 +168,11 @@ private static void populateWithDefaultHandlers(TypeHandlerLibrary serialization
 
         serializationLibrary.addTypeHandlerFactory(new TextureRegionAssetTypeHandlerFactory());
 
-        serializationLibrary.addTypeHandler(Vector4f.class, new LegacyVector4fTypeHandler());
-        serializationLibrary.addTypeHandler(Vector3f.class, new LegacyVector3fTypeHandler());
-        serializationLibrary.addTypeHandler(Vector2f.class, new LegacyVector2fTypeHandler());
-        serializationLibrary.addTypeHandler(Vector3i.class, new LegacyVector3iTypeHandler());
-        serializationLibrary.addTypeHandler(Vector2i.class, new LegacyVector2iTypeHandler());
```

Comment:
I'm PRETTY sure these can be safely removed. It is possible however, that they are still used to serialize objects containing TeraMath related members.